release/models/system/openconfig-system-controlplane.yang

```diff
@@ -199,6 +214,186 @@ module openconfig-system-controlplane {
               }
             }
           }
+          container built-in-limits {
```

Comment:
Suggest following a similar style of naming that is in the [integrated circuit drop counters](https://github.com/openconfig/public/blob/master/doc/Integrated-Circuit_pipeline_aggregated_counters_guide.md) which uses the term 'aggregated' to indicate there are likely lower level, more granular counters  which may optionally be exposed in in a [vendor provided model augment whose schema must follow a predefined pattern](https://github.com/openconfig/public/blob/master/doc/vendor_counter_guide.md).

```suggestion
            container drop {
```